Automattic/wp-calypso
# Plans: Ensure Jetpack plans have re-publicize feature

Ensure that Jetpack Premium and Business plans have re-publicized enabled.

Fixes #12784 

To test:
- Get this branch going at calypso.localhost:3000
- Select a Jetpack site with a Professional plan
- Go to posts/$site and click "Share" button under your blog post
- Ensure that you are not prompted to upgrade
